hometask_10/1/main.js

```diff
@@ -0,0 +1,7 @@
+document.addEventListener("DOMContentLoaded", function() { 
+  const img = document.querySelector('img');
+
+  console.log('Width: ', img.width);
+  console.log('Height: ', img.height);
+});
```

Comment:
it doesn't work
http://joxi.ru/L21b8vDSRD14jA
`DOMContentLoaded` doesn't give you guarantees image is loaded. Please, refer to [documentation](https://developer.mozilla.org/ru/docs/Web/Events/DOMContentLoaded) for more details